css/tokens/index.js

```diff
@@ -1,30 +1,24 @@
-const sassExtract = require('sass-extract');
 const fs = require('fs-extra');
 const path = require('path');
-const { convertSassVariable } = require('./conversion');
 const { quicktype, InputData, jsonInputForTargetLanguage } = require('quicktype-core');
+const { exporter } = require('sass-export');
 
 createTokens();
 
 async function createTokens() {
-	const indexFile = require.resolve('../src/tokens/index.scss');
+	const filePathStem = path.join(process.cwd(), './src/tokens');
+	const indexPath = path.resolve(filePathStem, 'index.scss');
 
-	/** @type {import('./types').SassConvertRendered }} **/
-	const rendered = await sassExtract.render({
-		file: indexFile
-	});
+	const filePaths = await getInputFilesFromIndex(filePathStem, indexPath);
+	checkFileComments(filePaths);
 
-	const collection = collectTokenSources(rendered);
+	const options = {
```

Comment:
How about `/** @type {{inputFiles: string[]}} */`?

Reply:
That will do in a pinch. The reason I was asking was because it would be awesome for developers have a better understanding of what the full list of options are - so a more full type would be good. We can see if we can get to that later.

Reply:
I think it would be most prudent to test all the options before we add the available options (since we found out that inputPaths doesn't actually work).

Reply:
I'm not sure what the harm would be in having a more accurate type here, but I agree that it isn't a priority now.

Reply:
Added a new type taken from Sass-export.